src/test/regress/sql/coordinator_shouldhaveshards.sql

```diff
@@ -214,13 +214,28 @@ INSERT INTO ref_table SELECT *, * FROM generate_series(1, 100);
 SELECT COUNT(*) FROM test JOIN ref_table USING(x);
 ROLLBACK;
 
+-- issue #4237
+CREATE TABLE test_append_table(a int);
+SELECT create_distributed_table('test_append_table', 'a', 'append');
+-- this will fail since it will try to create an empty placement in the
+-- coordinator as well
+SET citus.shard_replication_factor TO 3;
+SELECT master_create_empty_shard('test_append_table');
+-- this will create an empty shard with replicas in the two worker nodes
+SET citus.shard_replication_factor TO 2;
+SELECT master_create_empty_shard('test_append_table');
+-- verify groupid is not 0 for each placement
+SELECT COUNT(*) FROM pg_dist_placement WHERE shardid = 1503037 AND groupid > 0;
```

Comment:
The `shardid` will change when new tests are added. You can try to do a join between `pg_dist_placement` and `pg_dist_shard`.

Reply:
I thought new tests will be added below it. The shardid is an output of line 226 as well (so I put `1503037` in `.out` file) and I don't have an idea how to omit that shardid.

Reply:
Doing `SELECT 1 FROM master_create_empty_shard('test_append_table')` would fix for the line above.
And you could do something like this
```
SELECT COUNT(*) FROM pg_dist_placement p, pg_dist_shard s WHERE p.shardid=s.shardid AND s.logicalrelid = 'test_append_table'::regclass AND p.groupid > 0;
```

But I also saw some examples in the tests that set a new_shard_id parameter, maybe that could work too:
```
SELECT master_create_empty_shard('...') AS new_shard_id
\gset
```

Reply:
Makes sense, thanks!